### ISSUE TITLE
login links open in browser not webpin

### ISSUE THREAD

**Issue by seekr** [artemanufrij/webpin#111]
I'm trying to login to gmail from webpin window but it opens the login page in browser and does not go back to webpin to conclude the login.

I installed webpin from AppCenter

**Comment by giovannicaligaris** [comment]
I'm having a similar issue. If I need to login to a website using my google account, it will open on a different browser.

Maybe it can have a similar solution to Nativefier, which forces all windows to open from within the same browser.

**Comment by Justinzobel** [comment]
Still an issue.

**Comment by Justinzobel** [comment]
Output in case it's relevant.

`
justin@justin-desktop: ~
→ $ com.github.artemanufrij.webpin https://mail.google.com/mail/u/0/#inbox

(com.github.artemanufrij.webpin:9770): Gtk-CRITICAL **: 12:52:42.425: gtk_widget_set_sensitive: assertion 'GTK_IS_WIDGET (widget)' failed

(com.github.artemanufrij.webpin:9770): Gtk-CRITICAL **: 12:52:42.425: gtk_widget_set_sensitive: assertion 'GTK_IS_WIDGET (widget)' failed
Segmentation fault
justin@justin-desktop: ~
→ $ [9897:9897:0215/125246.713481:ERROR:chrome_content_client.cc(346)] Failed to locate and load the component updated flash plugin.
Opening in existing browser session.`

**Comment by janxkoci** [comment]
Cannot reproduce - https://gmail.com lets me login from within the webpin app window.

**Comment by Justinzobel** [comment]
https://youtu.be/O8Yz2hzMGOM - Can't upload videos here so posted to YouTube, I can replicate it every time.

**Comment by janxkoci** [comment]
So I don't get the intro screen, instead I'm shown directly the login screen. In the past it happened that Google was serving different users different version of website, especially login websites, as they run their experiments. Maybe the problem is with Google upstream..

**Comment by Justinzobel** [comment]
OK I found a hacky workaround, use
https://accounts.google.com/AccountChooser?service=mail&continue=https://mail.google.com/mail/
as
the URL. It just means you have to select your account each time you launch
the App which is not too bad.

It is laggy inside the webapp too for some reason.

**Comment by cedewey** [comment]
I'm running into the same problem with https://pomodor.app/timer

**Comment by janxkoci** [comment]
Yeah and I cannot login to Pocket with my Google account, as it opens the google login site in a browser. :confused:
